man/bootctl.xml

```diff
@@ -237,6 +237,19 @@
         cannot be written. Currently only applies to random seed operations.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--make-machine-id-directory=yes|no|auto</option></term>
+        <listitem><para>Control creation and deletion of <literal>$ESP/$MACHINE_ID</literal> during
+        <option>install</option> and <option>remove</option>, respectively.
+        <literal>auto</literal> is equivalent to to <literal>yes</literal> if <literal>/etc/machine-id</literal>
+        resides on a filesystem other than tmpfs and <literal>no</literal> otherwise.
+        Defaults to <literal>auto</literal>.</para>
+
+        <para>Overriding this may be desired to hide the machine ID from the (unencrypted) ESP,
+        configure a downstream generator like a <citerefentry><refentrytitle>kernel-install</refentrytitle><manvolnum>8</manvolnum></citerefentry> script, or,
```

Comment:
"configure a downstream generator" is unclear. `kernel-install` is neither downstream nor a generator...

Reply:
this very clearly says "like a kernel-install(8) script"

Reply:
Yeah, but in systemd "generator" has a specific meaning, and calling the script a generator would be confusing. And also "downstream" is not appropriate.

Reply:
Removed generator mention

Reply:
OK, so please push this later version.

Reply:
ah yeah, seems that I missed that. should be up now